SWOT-analysis.md

```diff
@@ -33,3 +33,14 @@
 
 - Consistently not baking good food items can result in losing existing customers. 
 
+# Our Business Persona
+
+- Our brand would embody the 'Explorer' archetype to depict the true French bakery experience. The goal is to take the customer to the trip to France with the help of famous food items.
+
+# Target customers:
+
+- Our brand would target customers with explorer type archetype who would want to experience the true french bakery taste without ever visiting France.
+
+- We would also target customers who are willing to pay a premium price for the Authentic french food experience. the goal is to convince the customer they don't have to spend thousands of dollars to visit france, when they can pay a nominal price (compared to actuallu visiting) have visit France through the their taste buds.
```

Comment:
compared to actually visiting?
our product is about going to physical store

Reply:
compared to actaully visiting france. they would still have to visit the store but it would be significantly cheaper than actually visiting france

Reply:
Then I think we should update the _compared to actuallu visiting_  to compared to actually france

Reply:
okay. creating a new pull request